shared/patches/expo-image-loader+4.3.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/expo-image-loader/android/build.gradle b/node_modules/expo-image-loader/android/build.gradle
+index 0959aa2..ee486cc 100644
+--- a/node_modules/expo-image-loader/android/build.gradle
++++ b/node_modules/expo-image-loader/android/build.gradle
+@@ -83,7 +83,7 @@ repositories {
+   mavenCentral()
+ }
+ 
+-def glideVersion = safeExtGet('glideVersion', '4.13.2')
++def glideVersion = safeExtGet('glideVersion', '4.16.0')
```

Comment:
fixes this issue which manifests itself as an 'context://' issue but its really a masked upstream exception